src/Symfony/Component/Form/Test/TestCaseSetUpTearDownTrait.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Test;
+
+use PHPUnit\Framework\TestCase;
+
+// Auto-adapt to PHPUnit 8 that added a `void` return-type to the setUp/tearDown methods
+
+if (method_exists(\ReflectionMethod::class, 'hasReturnType') && (new \ReflectionMethod(TestCase::class, 'tearDown'))->hasReturnType()) {
+    eval('
+    namespace Symfony\Component\Form\Test;
+
+    /**
+     * @internal
+     */
+    trait TestCaseSetUpTearDownTrait
+    {
+        private function doSetUp(): void
+        {
+        }
+
+        private function doTearDown(): void
+        {
+        }
+    
+        protected function setUp(): void
+        {
+            $this->doSetUp();
+        }
+        
+        protected function tearDown(): void
+        {
+            $this->doTearDown();
+        }
+    }
+');
+} else {
+    /**
+     * @internal
+     */
+    trait TestCaseSetUpTearDownTrait
+    {
+        /**
+         * @return void
```

Comment:
these docblocks should probably be removed

Reply:
I copied it from the other mentioned PR...

Reply:
The idea of these docblocks is to leverage https://github.com/symfony/symfony/pull/30323 and trigger deprecation notices when ppl don't add them.

Reply:
got it